src/AzureAuth.Test/CommandMainTest.cs

```diff
@@ -750,21 +750,34 @@ public void InteractiveAuth_IsEnabledIfEnvVarsAreNotSet()
             subject.InteractiveAuthDisabled().Should().BeFalse();
         }
 
+#if PlatformWindows
         [TestCase("non-empty-string")]
+        [Platform("Win")] // Only valid on Windows
         public void GetCombinedAuthMode_withInteractiveAuthDisabled(string noUser)
         {
             CommandMain subject = this.serviceProvider.GetService<CommandMain>();
             this.envMock.Setup(e => e.Get("AZUREAUTH_NO_USER")).Returns(noUser);
-            subject.GetCombinedAuthMode().Should().Be(AuthMode.IWA);
+            subject.CombinedAuthMode.Should().Be(AuthMode.IWA);
         }
 
+        [Platform("Win")] // Only valid on Windows

```

Comment:
If these are in an `#if PlatformWindows` then they are not compiled on other platforms, so we should not also need the Platform win attributes.